README.md

```diff
@@ -27,6 +27,8 @@ Register as a plugin, providing one or more of the following options:
   - `cookieName`: The name of the cookie.
   - `signed`: Indicates whether the cookie is signed or not. If set to `true`, the JWT will be verified using the unsigned value.
 
+Since this plugin is based on the [@fastify/jwt](https://www.npmjs.com/package/@fastify/jwt) `verifier`, it is also possibile to pass the options documented [here](https://github.com/fastify/fastify-jwt#verify) such as `cache` and `cacheTTL`.
```

Comment:
it might be worth providing an example, because it is not obvious how to provide those options. Do they need to be "prefixed" in some way, e.g. nested in a "verify" property or can they just be spread as root properties? Let's provide a minimal example to clarify